.mergify.yml

```diff
@@ -0,0 +1,21 @@
+queue_rules:
+  - name: default
+    conditions:
+      - "#approved-reviews-by>=1"
+      - -label~=(do-not-merge)
+      - status-success=terraform
+pull_request_rules:
+  - name: Automatic merge on approval and successful build
+    actions:
+      delete_head_branch: {}
+      queue:
+        method: squash
+        name: default
+        commit_message_template: |-
+          {{ title }} (#{{ number }})
+
+          {{ body }}
+    conditions:
+      - "#approved-reviews-by>=1"
+      - -label~=(do-not-merge)
+      - status-success=terraform
```

Comment:
Do we need to include `license/cla` or is that somehow included by default?
Else we'd need to always remember to not approve PRs that have no cla approval.